src/FlightMap/Widgets/CameraPageWidget.qml

```diff
@@ -202,6 +203,23 @@ Column {
                     anchors.left:   parent.left
                     anchors.right:  parent.right
                     spacing:        _margins
+                    Row {
+                        visible:    _isCamera
+                        spacing:    ScreenTools.defaultFontPixelWidth
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        QGCLabel {
+                            text:           qsTr("Camera Selector:")
+                            width:          _labelFieldWidth
+                            anchors.verticalCenter: parent.verticalCenter
+                        }
+                        QGCComboBox {
+                            id:             cameraSelector
+                            model:          _isCamera ? _dynamicCameras.cameraLabels : []
+                            width:          _editFieldWidth
+                            onActivated:    _dynamicCameras.cameras = index
```

Comment:
This throws an error - `qrc:/qml/CameraPageWidget.qml:219: TypeError: Cannot assign to read-only property "cameras"` 
Changing it to `_dynamicCameras.currentCamera = index` fixes it

Reply:
Yup, I had the same error someplace else (from copying/pasting from the original code) and missed this one. Things you miss when you can't test :P